### PR TITLE
chore(lint): add no only on precommit

### PR DESCRIPTION
**Pull request by canonbrother**
#### What this PR does / why we need it:
happy dev

annoying while forgot to remove then push
time wasted on CI
image

#### Which issue(s) does this PR fixes?:

Fixes #

### CODE REVIEW

**Comment by fuxingloh** [comment]
I think we might be able to accomplish this on eslint?

**Comment by fuxingloh** [comment]
Probably also a good idea to update the enforcement of testing with `toStrictEqual` 

> https://www.npmjs.com/package/eslint-plugin-jest might be useful

File: .eslintrc [hunk 1]
```diff
@@ -34,7 +35,9 @@
       }
     ],
     "import/no-default-export": "error",
-    "no-else-return": "error"
+    "no-else-return": "error",
+    "jest/no-disabled-tests": "error",
+    "jest/prefer-strict-equal": "error"
```
Comment (canonbrother) [inline .eslintrc:40]: FYI, it only restricts `toEqual` but `toBe`

**Review by fuxingloh — APPROVED** [review]

**Comment by fuxingloh** [comment]
These eslint changes highlight a few linting issues with our existing test, need to decide whether we should silent them or follow convention.

**Pull request closed without merging**